src/libs/three.module.js

```diff
@@ -30663,11 +30663,11 @@ function SphereBufferGeometry( radius, widthSegments, heightSegments, phiStart,
 
 		var uOffset = 0;
 
-		if ( iy == 0 && thetaStart == 0 ) {
```

Comment:
This line  was producing warnings, correct? I dont think you should modify this file, it is the "three js" devs job to maintain it.

Reply:
Yes and I did fix it, should I revert that commit?

Reply:
Yeah. Let this file intact. The change in there other ones are fine !!

Good work !